hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -624,16 +624,11 @@ public static Map<MetadataPartitionType, HoodieData<HoodieRecord>> convertMetada
   }
 
   private static List<HoodieRecord> getHoodieRecordsForLogFilesFromRollbackPlan(HoodieTableMetaClient dataTableMetaClient, String instantTime) {
-    /*List<HoodieInstant> instants = dataTableMetaClient.reloadActiveTimeline().filterRequestedRollbackTimeline()
-        .filter(instant -> instant.getTimestamp().equals(instantTime) && instant.isRequested()).getInstants();*/
-
     HoodieInstant rollbackInstant = new HoodieInstant(HoodieInstant.State.REQUESTED, HoodieTimeline.ROLLBACK_ACTION, instantTime);
-
-    // HoodieInstant rollbackInstant = instants.get(0);
     HoodieInstant requested = HoodieTimeline.getRollbackRequestedInstant(rollbackInstant);
     try {
       HoodieRollbackPlan rollbackPlan = TimelineMetadataUtils.deserializeAvroMetadata(
-          dataTableMetaClient.getActiveTimeline().readRollbackInfoAsBytes(requested).get(), HoodieRollbackPlan.class);
+          dataTableMetaClient.reloadActiveTimeline().readRollbackInfoAsBytes(requested).get(), HoodieRollbackPlan.class);
```

Comment:
Why is there a need to reload active timeline again? The `requested` instant is part of the rollback instant passed by the caller and data table metaclient is also passed by the caller. I think there should not be a need to reload.